examples/rbfe_single.py

```diff
@@ -120,31 +199,102 @@ def wrap_method(args, fxn):
     ordered_params = forcefield.get_ordered_params()
     ordered_handles = forcefield.get_ordered_handles()
 
-    gradient_clip_thresholds = {
-        nonbonded.AM1CCCHandler: 0.05,
-        nonbonded.LennardJonesHandler: np.array([0.001, 0])
-    }
+    handle_types_being_optimized = [AM1CCCHandler, LennardJonesHandler]
```

Comment:
We should have a way to avoid optimizing a particular component of the parameters, eg. for torsions, params are:

(k, phase, period), but we really dont want to optimize the period.

Reply:
Good point -- I should rework this. I haven't checked if the returned gradient w.r.t. discrete parameters like period is currently nan or 0, but we certainly will want to avoid updating discrete parameters like this when we start optimizing valence parameters.

I think a modular way to support masking by parameter type might be to zero out the relevant entries of the `search_direction`...